### DISCUSSION TITLE
Priority on enhancements

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#926] [Q&A]
Considering that bugs always come first, how do we organize for new features?
I personally have no preferences on the order to follow, so I let you decide

**Comment by csagan5** [comment]
We could use GitHub Projects to organise that; but since there is only the two of us, I do not think it is needed

**Reply by uazo** [reply]
ok, then i go in order, if you agree the next thing i try to do is the sync.
can you unblock the userscripts? if you want I have the arm64 apk ready to check in debug mode, or do you prefer the x86?

**Reply by csagan5** [reply]
You can do them in any order, since it is your free time and you must be inspired/motivated to do something for the project.

Importance-wise, #691 is probably the most important since a new user might not know about the adblock filters updater; next most important is #706 for security reasons, as not everyone might use F-Droid client to manage updates

**Reply by uazo** [reply]
perfect, was what I wanted to know, then I try first with #691

**Reply by csagan5** [reply]
As for the user scripts patch: let me try it after #905; no need for APKs, I cannot use them anyways for security reasons.
